Formula/pygmo.rb

```diff
@@ -0,0 +1,22 @@
+class Pygmo < Formula
+  desc "Scientific Python library for massively parallel optimization"
+  homepage "https://esa.github.io/pagmo2/"
+  url "https://github.com/esa/pagmo2/archive/v2.10.tar.gz"
+  sha256 "2fa95e2b464ddeadb9fc09bd314081293f02a1b6abc11c0b05064729a077227c"
+
+  depends_on "cmake" => :build
+  depends_on "boost-python3" => :build
+  depends_on "pagmo" => :build
```

Comment:
Why is the pagmo formula required to build pygmo ?

Reply:
@javian pagmo and pygmo are two separate projects. pagmo is a C++ library, pygmo is the set of python bindings for pagmo. pygmo depends on pagmo, because it needs the definitions of pagmo's functions and data structures in order to create the bindings.

Despite the fact that they are two separate projects, they are currently living within the same repository and source tree for historical reasons. This is something which can be a bit confusing. We will eventually split out pygmo completely from pagmo's source tree, but we are not at that point yet.

Reply:
Thanks for the explanation, yes it is a bit confusing but I do understand